crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -472,7 +472,7 @@ where
     ) -> Option<B256> {
         // check pre merge block error
         if insert_err.map(|err| err.is_block_pre_merge()).unwrap_or_default() {
-            return Some(B256::ZERO)
+            return Some(B256::ZERO);
```

Comment:
```suggestion
            return Some(B256::ZERO)
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145